utils/test_utils.ts

```diff
@@ -68,6 +70,27 @@ export function createJSONWebhookEvent(
   });
 }
 
+export function createJSONWebhookWebFormEvent(
+  event: string,
+  path: string,
+  payload: unknown,
+  pathParameters: KV,
+  queryStringParameters: KV,
+): APIGatewayProxyEventV2 {
+  return createAPIGatewayProxyEventV2("POST", path, {
+    headers: {
+      "Accept": "*/*",
+      "Content-Type": "application/x-www-form-urlencoded",
+      "User-Agent": "GitHub-Hookshot/f1aa6e4",
+      "X-GitHub-Delivery": "01b06e5c-d65c-11ea-9409-7e8b4a054eac",
+      "X-GitHub-Event": event,
+    },
+    data: btoa(JSON.stringify(payload)),
```

Comment:
I generated the four events we have in the testdata dir as `x-www-form-urlencoded`: https://gist.github.com/lucacasonato/81d7b03ee92f9172957e215508eb70dc. Can you add these to the testdata directory and use these as the body for the `x-www-form-urlencoded` tests instead of manually encoding the JSON ones?

(just add all four even though you only make use of two in the tests - we might need them later)